skt/executable.py

```diff
@@ -44,6 +45,58 @@ def full_path(path):
     return os.path.abspath(os.path.expanduser(path))
 
 
+def destroy_state_file(cfg):
+    """
+    Destroy the state file
+
+    Args:
+        cfg: A dictionary of skt configuration
+    """
+    state_file = cfg.get('state')
+    if os.path.isfile(state_file):
+        os.unlink(state_file)
+
+
+def read_state_file(cfg):
+    """
+    Read the state file from the disk
+
+    Args:
+        cfg: A dictionary of skt configuration
+    """
+    state_file = cfg.get('state')
+    if os.path.isfile(state_file):
+        with open(state_file, 'r') as fileh:
```

Comment:
We should account for a situation when the file doesn't exist / we don't have permission to open it etc

Reply:
That makes sense. I'll add that logic.